src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFHeaderLines.java

```diff
@@ -79,6 +79,8 @@ public static VCFFormatHeaderLine getEquivalentFormatHeaderLine(final String inf
         addFilterLine(new VCFFilterHeaderLine(VQSR_TRANCHE_SNP, "Temporary VQSLOD cutoff for SNPs until we implememnt full tranche sensitivities"));
         addFilterLine(new VCFFilterHeaderLine(VQSR_TRANCHE_INDEL, "Temporary VQSLOD cutoff for INDELs until we implememnt full tranche sensitivities"));
 
+        addFilterLine(new VCFFilterHeaderLine(EXCESS_HET_KEY, "Site has excess het value larger than the threshold"));
```

Comment:
I'm so surprised this wasn't there?  How was WARP dealing with this?

Reply:
maybe it was just putting it in the INFO annotation and VQSR was doing the thresholding?